dask_conversions/done/Dask_Coastline_Extraction_Landsat.ipynb

```diff
@@ -23,12 +23,10 @@
     "\n",
```

Comment:
<p>Here we seem to have lost data between 2013 and 2020. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SatelliteApplicationsCatapult/odc-hub/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>This may be a result of you not having complete set of scenes within your environment. </p>

Reply:
<p>I have found the source of this - please amend based on my other comment.</p>